python/cudf/cudf/core/column/column.py

```diff
@@ -82,7 +83,14 @@
 T = TypeVar("T", bound="ColumnBase")
 
 
-class ColumnBase(Column, Serializable, NotIterable):
+class ColumnBase(Column, Serializable, Reducible, NotIterable):
+    _VALID_REDUCTIONS = {
```

Comment:
it might be helpful for me if I were a new user to the codebase for a short comment to point me in the right direction, even just 

```suggestion
    
     # see mixin_factory.py
    _VALID_REDUCTIONS = {
```

Reply:
Would you want to see this in every class that inherits from Reducible? There are a lot of similar cases in cudf, e.g. at some point I added a note that `_constructor*` methods are for Dask and that `__array_*` functions are for numpy. Would you like to see similar comments in all places that those occur? I think that's basically the same issue as here and it would be helpful to standardize.